src/Database/Table/GroupedSelection.php

```diff
@@ -201,6 +201,10 @@ protected function loadRefCache()
 		}
 	}
 
+	protected function emptyResultSet($saveCache = TRUE, $deleteRererencedCache = TRUE) {
+		parent::emptyResultSet($saveCache, false);
```

Comment:
The reason of adding a parameter was that i was thinking e.g. about adding new descendant of `Selection`. Than this behaviour will be probably shared so i choose a parameter.

But i am completely fine with your solution too, if this is not an issue.